Cod3rsGrowth.Test/Singleton.cs

```diff
@@ -0,0 +1,60 @@
+﻿using Cod3rsGrowth.Dominio.Modelos;
+
+public class Singleton
+{
+    public sealed class SingletonTabelas
+    {
+        public static List<Carta> instanceCarta;
+        public static List<CopiaDeCartasNoBaralho> instanceCopiaDeCartasNoBaralho;
+        public static List<Baralho> instanceBaralho;
+        public static List<Jogador> instanceJogador;
+
+        public static List<Carta> GetInstanceCarta()
+        {
+            lock (typeof(List<Carta>))
+            {
+                if (instanceCarta == null)
+                {
+                    instanceCarta = new List<Carta>();
+                }
+            }
+            return instanceCarta;
+        }
+
+        public static List<CopiaDeCartasNoBaralho> GetInstanceCopiaDeCartasNoBaralho()
+        {
+            lock (typeof(List<CopiaDeCartasNoBaralho>))
+            {
+                if (instanceCopiaDeCartasNoBaralho == null)
+                {
+                    instanceCopiaDeCartasNoBaralho = new List<CopiaDeCartasNoBaralho>();
+                }
+            }
+            return instanceCopiaDeCartasNoBaralho;
+        }
+
+        public static List<Baralho> GetInstanceBaralho()
+        {
+            lock (typeof(List<Baralho>))
+            {
+                if (instanceBaralho == null)
+                {
```

Comment:
utilizar um ternário para remover esse if, algo tipo assim 

instanceBaralho = instanceBaralho is null ? new List<Baralho>() : instanceBaralho 

pode utilizar em todas esse padrão